### PR TITLE
FIX close: 1535 Add a button to hide the empty service on console at service list page

### PR DESCRIPTION
**Pull request by zhaoyuguang**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

Add a button to hide the empty service on console
在服务列表页面，增加一个按钮可以隐藏空服务
页面效果：
image

https://github.com/alibaba/nacos/issues/1535

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

**Comment by zhaoyuguang** [comment]
@nkorange  hi, I`m ready for this pr.

File: naming/src/main/java/com/alibaba/nacos/naming/core/ServiceManager.java [hunk 1]
```diff
@@ -637,14 +638,22 @@ private void putServiceAndInit(Service service) throws NacosException {
...
+        if (StringUtils.isBlank(regex) && !hasIpCount) {
+            return new ArrayList<>(chooseServiceMap(namespaceId).values());
+        }
         List<Service> result = new ArrayList<>();
         for (Map.Entry<String, Service> entry : chooseServiceMap(namespaceId).entrySet()) {
             Service service = entry.getValue();
             String key = service.getName() + ":" + ArrayUtils.toString(service.getOwners());
-            if (key.matches(regex)) {
-                result.add(service);
+            if (StringUtils.isBlank(regex) || !key.matches(regex)) {
```
Comment (nkorange) [inline naming/src/main/java/com/alibaba/nacos/naming/core/ServiceManager.java]: Should be:
```java
            if (StringUtils.isNotBlank(regex) && !key.matches(regex)) {
                continue;
            }
```

Comment (zhaoyuguang) [reply naming/src/main/java/com/alibaba/nacos/naming/core/ServiceManager.java]: This case: when `Service Name is null` and `ipCount>0`,
I still take him to the outer layer, so the code looks very clear

**Comment by zhaoyuguang** [comment]
Two conditions four cases
image
image
image
image

**Pull request merged**